### PR TITLE
Correct passing ssl params

### PR DESCRIPTION
**Pull request by SalnikovIgor**

### CODE REVIEW

File: starting_package/openvino_workbench/docker_config_creator.py [hunk 1]
```diff
@@ -148,6 +148,7 @@ def create_config_for_container(passed_arguments: Namespace) -> dict:
                               'PYTHON_WRAPPER': 1},
               'name': passed_arguments.container_name,
               'ports': {INTERNAL_PORT: (passed_arguments.ip, passed_arguments.port)},
+              'hostname': 'workbench',
```
Comment (SalnikovIgor) [inline starting_package/openvino_workbench/docker_config_creator.py:151]: Need to check this parameter

**Review by akashchi — APPROVED** [review]

File: starting_package/openvino_workbench/docker_config_creator.py (hunk 1 above)
Comment (akashchi) [reply starting_package/openvino_workbench/docker_config_creator.py]: Are you done with checks?

Comment (SalnikovIgor) [reply starting_package/openvino_workbench/docker_config_creator.py]: Yes, everything pass well with this parameter.

**Comment by akashchi** [comment]
@artyomtugaryov please, take a look.

**Review by artyomtugaryov — APPROVED** [review]

**Pull request merged**